### PR TITLE
[Acceptance] Fix issues with tokenized `clouds.yaml`

### PR DESCRIPTION
**Pull request by lego963**
## Summary of the Pull Request

* Remove `OS_AUTH_URL` env variable check from required variables in acceptance tests.
* Set `Region` in client generation function.

## PR Checklist

* [x] Refers to: #1291
* [x] Tests added/passed.

## Acceptance Steps Performed

```
=== RUN   TestAccASV1Policy_basic
--- PASS: TestAccASV1Policy_basic (72.05s)
PASS

Process finished with the exit code 0


=== RUN   TestAccVpcV1_basic
--- PASS: TestAccVpcV1_basic (60.38s)
=== RUN   TestAccVpcV1_update
--- PASS: TestAccVpcV1_update (93.83s)
=== RUN   TestAccVpcV1_timeout
--- PASS: TestAccVpcV1_timeout (56.55s)
PASS

Process finished with the exit code 0
```

### CODE REVIEW

File: releasenotes/notes/acceptance-region-9431491e16222211.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    **[Acceptance]** Set ``Region`` while client is generating (`#1298 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1298>`_)
```
Comment (outcatcher) [inline releasenotes/notes/acceptance-region-9431491e16222211.yaml]: The change affects not only acceptance tests. Better to say that you read effective region from the IAM. Also that doesn't seem to affect user experience, so I don't think that RN is required here

File: .zuul.yaml [hunk 2]
```diff
@@ -20,6 +20,9 @@
         - tflint
         - goreleaser-build:
             nodeset: fedora-32-large
+    check-post:
+      jobs:
+        - golang-make-functional
```
Comment (outcatcher) [inline .zuul.yaml]: Let's make it in another PR

File: GNUmakefile [hunk 3]
```diff
@@ -62,4 +62,4 @@ tffmtfix: tools
 	@echo "==> Fixing docs terraform blocks code with terrafmt..."
 	@find ./docs -type f -name "*.md" | sort | while read f; do terrafmt fmt $$f; done
 
-.PHONY: build test testacc vet fmt fmtcheck errcheck test-compile tflint tffmtfix lint
+.PHONY: build test acceptance vet fmt fmtcheck errcheck test-compile tflint tffmtfix lint
```
Comment (outcatcher) [inline GNUmakefile]: Same as for changes in `.zuul.yaml`

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: GNUmakefile (hunk 3 above)
Comment (lego963) [reply GNUmakefile]: done

File: .zuul.yaml (hunk 2 above)
Comment (lego963) [reply .zuul.yaml]: done

File: releasenotes/notes/acceptance-region-9431491e16222211.yaml (hunk 1 above)
Comment (lego963) [reply releasenotes/notes/acceptance-region-9431491e16222211.yaml]: done

**Review by anton-sidelnikov — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]
